### DISCUSSION TITLE
docs.scrypted.app

### DISCUSSION THREAD

**Discussion by p-taylor** [koush/scrypted#1103] [Q&A] [answered]
Hi there

Can anyone tell me what platform / template scrypted are using for their documentation site at [docs.scrypted.app](url)?

**Comment by eox2** [comment, marked as answer]
Looks like Vitepress

https://github.com/vuejs/vitepress

https://github.com/vuejs/vitepress/tree/main/docs

https://vitepress.dev/guide/getting-started

**Reply by p-taylor** [reply]
Thank you – Much appreciated
